action.yml

```
@@ -16,6 +16,10 @@ inputs:
     description: 'Skip PHP Setup (useful if it was setup in another step)'
     required: false
     default: 'false'
+  extensions:
+    descriptions: 'Files only with this extension(s) will be tested'
+    required: false,
+    default: ''
```

Probably worth passing `php` as a default here rather than an empty string.

```suggestion
    default: 'php'
```

---

My reasoning behind making the default empty: if we don't supply `--extensions` to `phpcs` in the command in `entrypoint.sh` - it checks at least js and css files alongside php files. I'm not exctly sure if other files extensions are included alongside php, js, and css. So, if we put php as the default to this input, the default behaviour of this github action will change. If you're fine with that, we can make `php` the default. It'll make the shell command simpler as well.

---

Given that this is specifically a _phpcompatibility_ action, I'm okay with defaulting to PHP. I don't see a good reason for files that _aren't_ `.php` to be checked by an action that explicitly says PHP Compatibility (and runs a PHPCS ruleset), except in the case that those files might have different file extensions (but are still written in PHP).

---

Great! That's the reason I made this change too! I was getting warnings from my js files because they were apparently minified and I didn't really need my js files checked by this.